src/common/hostname.h

```diff
@@ -16,7 +16,11 @@
 #define CEPH_COMMON_HOSTNAME_H
 
 #include <string>
+#include <cassert>
 
-extern std::string ceph_get_hostname();
-extern std::string ceph_get_short_hostname();
+#include <unistd.h>
+#include <errno.h>
```

Comment:
i don't think you are using any errno in this change.

Reply:
yeah i think so.